src/Paprika/Store/IBatchContext.cs

```diff
@@ -72,6 +72,8 @@ Page EnsureWritableCopy(ref DbAddress addr)
     Page TryGetPageAlloc(ref DbAddress addr, PageType pageType);
 
     BatchStats? Stats { get; }
+
+    public unsafe Page NullPage => new Page((byte*)0);
```

Comment:
If we pass null pointer here, some things may blow up. Which might be ok, but I think that the preffered way of doing it, would be to have something that can be naturally retrieved by `.GetAddress`. Having that said, I'd go with

```suggestion
    public unsafe Page NullPage => new Page(GetAt(DbAddress.Null));
```